docs/dev/adding_instruments/instrument.rst

```diff
@@ -152,14 +152,15 @@ To cite the `Python documentation <https://docs.python.org/3.11/howto/enum.html>
 
     An Enum is a set of symbolic names bound to unique values. They are similar to global variables, but they offer a more useful :code:`repr()`, grouping, type-safety, and a few other features.
 
-As our signal values are often integers, the most appropriate enum types are :code:`IntEnum` and :code:`IntFlag`.
+An Enum can be combined with a type of value that you want to bind symbolic names to, by multiple inheritance of the type and :code:`Enum` (or :code:`Flag`).
+As our signal values are often integers, the most appropriate combinations are :code:`(int, Enum)` and :code:`(int, Flag)`.
```

Comment:
```suggestion
As our signal values are often integers, the most appropriate combinations are :code:`(int, Enum)` and :code:`(int, Flag)`.
This way, we get meaningful information when printing (e.g. `InstrMode.HEATING`), instead of plain integers (`1`).
```
maybe illuminate _why_ we prefer this?

Reply:
This is no longer relevant I think.